<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="griffon.exceptions.ArtifactNotFoundExceptionSpec" tests="3" skipped="0" failures="0" errors="0" timestamp="2017-04-28T12:49:47" hostname="testing-docker-929acd7b-76bf-4253-9fc4-85722f6f076e" time="0.005">
  <properties/>
  <testcase name="Exception should have proper formatted message[0]" classname="griffon.exceptions.ArtifactNotFoundExceptionSpec" time="0.004"/>
  <testcase name="Exception should have proper formatted message[1]" classname="griffon.exceptions.ArtifactNotFoundExceptionSpec" time="0.0"/>
  <testcase name="Exception should have proper formatted message[2]" classname="griffon.exceptions.ArtifactNotFoundExceptionSpec" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
